src/locale/__tests__/translations.spec.ts

```diff
@@ -9,6 +9,7 @@ describe('translations', () => {
     beforeEach(() => {
       translator = {
         get: () => 'SNTable.Accessibility.RowsAndColumns',
+        language: () => 'someLanguage',
```

Comment:
is this part of offboarding the flag?

Reply:
Nope, that was my changes from nebula => causing types error because I linked it already, ✅ fixed now